modules/core/src/test/java/org/apache/ignite/util/GridCommandHandlerClusterByClassTest.java

```diff
@@ -150,20 +151,72 @@
     /** */
     private File customDiagnosticDir;
 
+    /** */
+    private static final int serverNodeCnt = 2;
+    /** */
+    private static final int clientNodeCnt = 1;
+
+    /** */
+    protected static IgniteEx crd;
+    /** */
+    private static IgniteEx client;
+
+    /** {@inheritDoc} */
+    @Override protected void beforeTestsStarted() throws Exception {
+        crd = startGrids(serverNodeCnt);
+        client = startGrid("client");
+
+        crd.cluster().active(true);
+    }
+
+    /** {@inheritDoc} */
+    @Override protected void afterTestsStopped() throws Exception {
+        GridTestUtils.cleanIdleVerifyLogFiles();
+
+        try (DirectoryStream<Path> files = newDirectoryStream(Paths.get(U.defaultWorkDirectory()), this::idleVerifyRes)) {
+            for (Path path : files)
+                delete(path);
+        }
```

Comment:
I think you should override `cleanPersistenceDir()` method in `GridCommandHandlerAbstractTest` and add removing dump files

Reply:
Fix it